charts/clickhouse/values.yaml

```diff
@@ -528,6 +564,42 @@ clickhouseOperator:
   # -- TopologySpreadConstraints describes how Clickhouse Operator pods ought to spread
   topologySpreadConstraints: []
 
+  ## Configure liveness and readiness probes.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+  ##
+  startupProbe:
+    enabled: true
+    port: metrics
+    path: /chi
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  livenessProbe:
+    enabled: true
+    port: metrics
+    path: /chi
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+  readinessProbe:
+    enabled: true
+    port: metrics
+    path: /chi
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    timeoutSeconds: 5
+    failureThreshold: 6
+    successThreshold: 1
+
+  ## Custom startup, liveness and readiness probes
+  customStartupProbe: {}
+  customLivenessProbe: {}
+  customReadinessProbe: {}
+
```

Comment:
Why is this needed?
```suggestion
  ## Custom startup, liveness and readiness probes
  customStartupProbe: {}
  customLivenessProbe: {}
  customReadinessProbe: {}

```

Reply:
Giving flexibility to users to update to any custom logic - it could be running some shell script for the health check.

Reply:
They can just override `  startupProbe`, `livenessProbe`. Why need a custom one?

Reply:
We will need another flag for checking if the probe type is HTTP, TCP, Exec, or gRPC.

The default ones can fit well with beginners - they can do minimal changes with the rest recommended defaults.

For the advanced users who may want to try different probe types, we would give them more flexibility.

Reply:
If that is the case then just do:
livenessProbe:
{{ toYaml livenessProbe }}

and let users pass the configuration from the values file.

It's complicating the setup which is seems unnecessary.